### DISCUSSION TITLE
Is this supposed to handle the System Tray / Taskbar as well?

### DISCUSSION THREAD

**Discussion by Rambetino** [manutalcual/winredock#51] [General]
I just installed winredock on my Windows 10 machine running 4 monitors. The screens are old and use a lot of power so I tend to shut them down when not in use. Naturally, this throws off the display arrangement that this program so perfectly fixes! Thanks for that !!

The only remaining issue is that my System Tray / Taskbar I keep off to one screen and double the height. I have to re-do this every time I power cycle the screens if the machine is still running. Is winredock able to change that to a previous state as well or is that outside of the scope of what this can do?

**Comment by manutalcual** [comment]
Hi. Thank you for your kind words.
No. The tool isn't supposed to do that and unfortunately I don't have time just now to do anything with that.
Plus, I use mainly Linux so I don't have any machine to develop and test it now. Sorry.
BTW, I think Windows 11 remembers windows positions now.

**Reply by siriusgithub** [reply]
Yes WIN 11 supports this natively now.
